frontend/src/utils/metadata.ts

```diff
@@ -0,0 +1,107 @@
+import type { AudioDetail, ImageDetail, Metadata } from "~/types/media"
+import { IMAGE } from "~/constants/media"
+
+import type { NuxtI18nInstance } from "@nuxtjs/i18n"
+
+const getImageType = (
+  imageType: string | undefined,
+  i18n: NuxtI18nInstance
+) => {
+  if (imageType) {
+    if (imageType.split("/").length > 1) {
+      return imageType.split("/")[1]
+    }
+    return imageType
+  }
+  return i18n.t("mediaDetails.information.unknown")
+}
+
+const getAudioType = (audio: AudioDetail, i18n: NuxtI18nInstance) => {
+  if (!audio.alt_files)
+    return audio.filetype ?? i18n.t("mediaDetails.information.unknown")
+  const altFormats = audio.alt_files.map((altFile) => altFile.filetype)
+  if (audio.filetype) {
+    altFormats.unshift(audio.filetype)
```

Comment:
The subsequent conversion to `Set` (unordered) and back to `Array` negates the use of `unshift` to insert at the start of the array.

Reply:
Great catch! I'll update this.